SonicWall/sonicwall-fw/tests/test_file_transfer_result.json

```diff
@@ -0,0 +1,83 @@
+{
+  "input": {
+    "sekoiaio": {
+      "intake": {
+        "dialect": "SonicWall Firewall",
+        "dialect_uuid": "ee0b3023-524c-40f6-baf5-b69c7b679887"
+      }
+    },
+    "message": "CEF:0|SonicWall|NSa 4700|7.0.1-5000-R3000|1460|Capture ATP File Transfer Result|5|cat=1 gcat=3 src=10.0.10.20 spt=444 deviceInboundInterface=X6-V333 dst=123.45.67.890  dpt=123 deviceOutboundInterface=X1 proto=tcp/12345 in=1500 msg=\"Gateway Anti-Virus Status: SMB file restart detected. File forwarding to Sandbox truncated for filename: hello.xlsx.\" cnt=123456 fw_action=\"NA\" fileid=\"0b9999999999999999ff99\" filetxstatus=230"
+  },
+  "expected": {
+    "message": "CEF:0|SonicWall|NSa 4700|7.0.1-5000-R3000|1460|Capture ATP File Transfer Result|5|cat=1 gcat=3 src=10.0.10.20 spt=444 deviceInboundInterface=X6-V333 dst=123.45.67.890  dpt=123 deviceOutboundInterface=X1 proto=tcp/12345 in=1500 msg=\"Gateway Anti-Virus Status: SMB file restart detected. File forwarding to Sandbox truncated for filename: hello.xlsx.\" cnt=123456 fw_action=\"NA\" fileid=\"0b9999999999999999ff99\" filetxstatus=230",
+    "event": {
+      "code": "1460",
+      "severity": 5,
+      "category": [
+        "network"
+      ],
+      "kind": "event",
+      "reason": "Gateway Anti-Virus Status: SMB file restart detected. File forwarding to Sandbox truncated for filename: hello.xlsx.",
```

Comment:
I think we can extract the filename from the message with a grok stage and add it to `file.name`